test/unit/specs/FontStyle.spec.js

```diff
@@ -0,0 +1,38 @@
+import Vue from 'vue'
+import FontStyle from '@/components/FontStyle'
+
+// Test number of font colors showing
+describe('FontStyle.vue', () => {
+  it('Font Colors should be 11', () => {
+    const Constructor = Vue.extend(FontStyle)
+    const vm = new Constructor().$mount()
+    expect(vm.fontColors.length).to.equal(11)
```

Comment:
This should b `vm.$data.fontColor...` or something line that, we should be using data

Reply:
I thought that was supposed to be for functions, test cases or PR added after the testing tutorial on wednesday?